### PR TITLE
feat(git-token): add services count

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-896

Capture d’écran 2024-03-07 à 12 19 39

Capture d’écran 2024-03-15 à 15 55 06

Capture d’écran 2024-03-15 à 15 56 23

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> The backlog never ends, be at peace with it

File: libs/shared/ui/src/lib/components/accordion/accordion.tsx [hunk 1]
```diff
@@ -49,14 +49,14 @@ const AccordionContent = forwardRef<ElementRef<typeof AccordionPrimitive.Content
     <AccordionPrimitive.Content
       className={twMerge(
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/accordion/accordion.tsx:50]: I removed the children wrapper to allow the possibility to override every class name

File: libs/domains/organizations/feature/src/lib/git-token-list/git-token-list.spec.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
+import * as utilDates from '@qovery/shared/util-dates'
 import { renderWithProviders } from '@qovery/shared/util-tests'
 import GitTokenList from './git-token-list'
 
+// Importez toutes les fonctions du module
```
Comment (ctjhoa) [inline libs/domains/organizations/feature/src/lib/git-token-list/git-token-list.spec.tsx:5]: to remove / translation

**Review by ctjhoa — CHANGES_REQUESTED** [review]

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**